com.unity.cinemachine/Tests/Runtime/CamerasBlendingTests.cs

```diff
@@ -40,97 +37,80 @@ public override void SetUp()
             m_Target.enabled = false;
             m_Source.transform.position = Vector3.zero;
             m_Target.transform.position = new Vector3(10, 0, 0);
-
-            base.SetUp();
-            
-            m_Brain.UpdateMethod = CinemachineBrain.UpdateMethods.ManualUpdate; 
         }
 
         [UnityTest]
         public IEnumerator BlendFromSourceToTarget()
         {
             // Check that source vcam is active
-            m_Brain.ManualUpdate();
+            yield return UpdateCinemachine();
             Assert.That(ReferenceEquals(m_Brain.ActiveVirtualCamera, m_Source));
-            yield return null;
             
             // Active target and blend from source to target completely
             m_Target.enabled = true;
-            var timeElapsed = 0f;
-            m_Brain.ManualUpdate();
-            timeElapsed += CinemachineCore.UniformDeltaTimeOverride;
-            while (timeElapsed < k_BlendingTime)
+            var startTime = CinemachineCore.CurrentTimeOverride;
```

Comment:
Since it's the job of `CinemachineRuntimeTimeInvariantFixtureBase` to manage current time, I would add a `CurrentTime` accessor to it, so clients don't need to know the implementation details

Reply:
Building on that idea, you can morph the API into something that is frame-based.  Instead of `UpdateCinemachine` you can have `WaitOneFrame`.  You can add variations that control how many frames to wait, and how many physics frames to wait.  Then, clients reason in terms of frames and current time, as managed by `CinemachineRuntimeTimeInvariantFixtureBase`.